zebra-rpc/Cargo.toml

```diff
@@ -43,6 +43,7 @@ proptest = { version = "0.10.1", optional = true }
 proptest-derive = { version = "0.3.0", optional = true }
 
 [dev-dependencies]
+insta = { version = "1.14.0", features = ["ron", "redactions"] }
```

Comment:
Test Coverage: Can we use JSON snapshots instead?

`lightwalletd` uses the JSON output, not the Rust structs, so that's what we want to snapshot. (Also, there are ways to change the JSON output without changing the Rust structs, and we want to catch those.)

Reply:
https://github.com/ZcashFoundation/zebra/pull/4352/commits/2e6f9f4de3faf42c17ccf8f7356dd838422ee5cf